issues/issue_3.md

```diff
@@ -87,8 +87,15 @@ Specifying multiple outputs is as simple as listing many strings separated by co
 You can specify multiple inputs or parameters in exactly the same way.
 Add the code above to the Snakefile.
 
-> NOTE: The path to each output file is included.
+Snakemake stores everything specified in the `input`, `output`, and `params` directives as Python list-like objects.
+That means that you can treat the object `snakemake.output` as a regular Python list, for example.
```

Comment:
What do you think of relating this back to how we called them by name before, and now we're suggesting that you can call by index?
```suggestion
That means that you can treat the object `snakemake.output` as a regular Python list, for example. You are able to call items in the Snakemake directive by index (not just by name, like we did before).
```

Reply:
I like it!